hotsos/cli.py

```diff
@@ -314,11 +314,12 @@ def cli(data_root, version, defs_path, templates_path, all_logs, quiet,
         data_root = fix_data_root(data_root)
 
         if is_snap() and data_root == '/':
-            print("WARNING: hotsos is installed as a snap which only "
+            print("ERROR: hotsos is installed as a snap which only "
                   "supports running against a sosreport due to access "
-                  "restrictions. If you want to analyse a host you need to "
+                  "restrictions.\n\n"
+                  "If you want to analyse a host you need to "
                   "use an alternative installation method e.g. debian "
-                  "package - see https://github.com/canonical/hotsos#install "
+                  "package - see docs at https://github.com/canonical/hotsos "
```

Comment:
Should we rather point to the readthedocs docs?

Reply:
i mean we could i just thought is more future proof in case the url ever changed

Reply:
you are right, ill change it